mxp/mxpOpenAtomic.cpp

```
@@ -24,14 +24,47 @@ static char BASED_CODE THIS_FILE[] = __FILE__;
+  }
+
+class CScopedMXPAction
+  {
+  public:
+    CScopedMXPAction (CAction * pAction) : m_pAction (pAction) { }
+    ~CScopedMXPAction ()
+      {
+      if (m_pAction)
+        m_pAction->Release ();
+      }
+    CAction * Release ()
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)